backend/inspirehep/records/marshmallow/literature/ui.py

```diff
@@ -129,11 +129,12 @@ def get_arxiv_fulltext_link(self, data):
         return missing
 
     def get_internal_fulltext_link(self, data):
+        # Assume all non hidden are fulltext - INSPIR-3268
```

Comment:
I don't think this is needed as we can trace back the change to the commit and you have the JIRA task in the commit message

Reply:
:+1: